python/parser.js

```diff
@@ -93,11 +95,8 @@ function region(region_param, first_call) {
     }
 }
 
-function ignoreRegions(arr) {
-    return arr.reduce((acc, val) => `${acc}.ignore(${regionParameter(val)})`, '')
-}
-function layoutRegions(arr){
-    return arr.reduce((acc, val) => `${acc}.layout(${regionParameter(val)})`, '')
+function regions(kind, arr) {
+    return arr.reduce((acc, val) => `${acc}.${kind}(${regionParameter(val)})`, '')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    return arr.reduce((acc, val) => `${acc}.${kind}(${regionParameter(val)})`, "")
```